lastuser_ui/forms/client.py

```diff
@@ -176,14 +176,14 @@ class UserPermissionAssignForm(Form):
     """
     Assign permissions to a user
     """
-    username = wtforms.TextField(__("User"), validators=[wtforms.validators.Required()],
-        description=__("Lookup a user by their username or email address"))
+    username = UserSelectField(__("User"), validators=[wtforms.validators.Required()],
+        description=__("Lookup a user by their username or email address"), lastuser=None, usermodel=User)
     perms = wtforms.SelectMultipleField(__("Permissions"), validators=[wtforms.validators.Required()])
 
     def validate_username(self, field):
-        existing = getuser(field.data)
+        existing = getuser(field.data.name)
```

Comment:
You shouldn't need to call `getuser` since `field.data` is already the user account. As long as it's not `None`, you have a user.